<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSelectorDialog</class>
 <widget class="QDialog" name="DeviceSelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device Selector Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Ethernet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Device Selection and Arguments</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditArgs"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_configureButton">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Scheduling</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Update Period (ms)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="m_updatePeriodSpinbox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Serial">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Device Documentation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEditDoc">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeviceSelectorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeviceSelectorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
